Sources/Twilio/Twilio.swift

```diff
@@ -1,80 +1,65 @@
 import Vapor
 
-public protocol TwilioProvider: Service {
-    var accountId: String { get }
-    var accountSecret: String { get }
-    func send(_ sms: OutgoingSMS, on container: Container) throws -> Future<Response>
+public protocol TwilioProvider {
+    func send(_ sms: OutgoingSMS) -> EventLoopFuture<ClientResponse>
 }
 
-// MARK: - Engine
 public struct Twilio: TwilioProvider {
-    public enum Error: Debuggable {
-        /// Encoding problem
-        case encodingProblem
-
-        /// Identifier
-        public var identifier: String {
-            switch self {
-            case .encodingProblem:
-                return "twilio.encoding_error"
-            }
-        }
-
-        /// Reason
-        public var reason: String {
-            switch self {
-            case .encodingProblem:
-                return "Encoding problem"
-            }
-        }
+    let application: Application
+    
+    public init (_ app: Application) {
+        application = app
     }
+}
 
-    /// Error response object
-    public struct ErrorResponse: Decodable {
+// MARK: - Configuration
 
-        /// Error messsage
-        public let message: String
+extension Twilio {
+    struct ConfigurationKey: StorageKey {
+        typealias Value = TwilioConfiguration
     }
 
-    public var accountId: String
-    public var accountSecret: String
-
-
-    /// Initializer
-    ///
-    /// - Parameters:
-    ///   - accountId: Account ID provided by Twilio
-    ///   - accountSecret: Account secret provided by Twilio
-    public init(accountId: String, accountSecret: String) {
-        self.accountId = accountId
-        self.accountSecret = accountSecret
+    public var configuration: TwilioConfiguration? {
```

Comment:
Does this need to be public?

Reply:
Yes it should be public to configure it by user as it shown in readme
```swift
import Twilio

// Called before your application initializes.
func configure(_ app: Application) throws {
    /// case 1
    /// put into your environment variables the following keys:
    /// TWILIO_ACCOUNT_ID=...
    /// TWILIO_ACCOUNT_SECRET=...
    app.twilio.configuration = .environment

    /// case 2
    /// manually
    app.twilio.configuration = .init(accountId: "<account id>", accountSecret: "<account secret>")
}
```

Reply:
Thanks for the example :) makes sense.